BedrockServer.cpp

```diff
@@ -1609,11 +1599,8 @@ void BedrockServer::suppressCommandPort(const string& reason, bool suppress, boo
     if (suppress) {
         // Close the command port, and all plugin's ports. Won't reopen.
         SHMMM("Suppressing command port");
-        if (!portList.empty()) {
-            closePorts({_controlPort});
-            _portPluginMap.clear();
-            _commandPort = nullptr;
-        }
+        _commandPort = nullptr;
```

Comment:
Since _commandPort is now a `unique_ptr`, it deletes the object it points at when unassigned, which closes the underlying socket. Same for the ports contained in `_portPluginMap`.